pkg/controller/util.go

```diff
@@ -420,6 +467,7 @@ func MakeImporterPodSpec(image, verbose, pullPolicy string, podEnvVar *importPod
 							Protocol:      v1.ProtocolTCP,
 						},
 					},
+					Resources: *resourceRequirements,
```

Comment:
This is a potential null pointer, I would add a section below like this:
```go
if resourceRequirements != nil {
  pod.Spec.Containers[0].Resources = *resourceRequirements
}
```
This way we will only add it, if we now that it is not nil. and we will only dereference it if we know it is not nil.

Reply:
If we use cdiconfig.status.PodResourceRequirements, which is always be fully populated, then *resourceRequirements will not be a null pointer, right?

Reply:
Since we cannot guarantee that whomever calls MakeImporterPodSpec will always set that to something, IMO its better to be a bit defensive and deal with the case where someone didn't set it. But you are right in the context of this patch, that should never be nil.

Reply:
DONE